main.go

```diff
@@ -4,55 +4,224 @@ import (
+			EnvVar: "EMAIL_PASSWORD,PLUGIN_PASSWORD",
+		},
+		cli.BoolFlag{
+			Name:   "skip.verify",
+			Usage:  "skip tls verify",
+			EnvVar: "PLUGIN_SKIP_VERIFY",
+		},
+		cli.StringSliceFlag{
+			Name:   "recipients",
+			Usage:  "recipient addresses",
+			EnvVar: "EMAIL_RECIPIENTS,PLUGIN_RECIPIENTS",
+		},
+		cli.BoolFlag{
+			Name:   "recipients.only",
+			Usage:  "send to recipients only",
+			EnvVar: "PLUGIN_RECIPIENTS_ONLY",
+		},
+		cli.StringFlag{
+			Name:   "template.subject",
+			Value:  DefaultSubject,
+			Usage:  "subject template",
+			EnvVar: "PLUGIN_SUBJECT",
+		},
+		cli.StringFlag{
+			Name:   "template.body",
+			Value:  DefaultTemplate,
+			Usage:  "body template",
+			EnvVar: "PLUGIN_BODY",
+		},
 
-	if vargs.Template == "" {
-		vargs.Template = defaultTemplate
-	}
+		// Drone environment
+		cli.StringFlag{
+			Name:   "repo.owner",
+			Usage:  "repository owner",
+			EnvVar: "DRONE_REPO_OWNER",
+		},
+		cli.StringFlag{
+			Name:   "repo.name",
+			Usage:  "repository name",
+			EnvVar: "DRONE_REPO_NAME",
+		},
+		cli.StringFlag{
+			Name:   "commit.sha",
+			Usage:  "git commit sha",
+			EnvVar: "DRONE_COMMIT_SHA",
+		},
+		cli.StringFlag{
+			Name:   "commit.ref",
+			Value:  "refs/heads/master",
+			Usage:  "git commit ref",
+			EnvVar: "DRONE_COMMIT_REF",
+		},
+		cli.StringFlag{
+			Name:   "commit.branch",
+			Value:  "master",
+			Usage:  "git commit branch",
+			EnvVar: "DRONE_COMMIT_BRANCH",
+		},
+		cli.StringFlag{
+			Name:   "commit.author.name",
+			Usage:  "git author name",
+			EnvVar: "DRONE_COMMIT_AUTHOR",
+		},
+		cli.StringFlag{
+			Name:   "commit.author.email",
+			Usage:  "git author email",
+			EnvVar: "DRONE_COMMIT_AUTHOR_EMAIL",
+		},
+		cli.StringFlag{
+			Name:   "commit.author.avatar",
+			Usage:  "git author avatar",
+			EnvVar: "DRONE_COMMIT_AUTHOR_AVATAR",
+		},
+		cli.StringFlag{
+			Name:   "commit.message",
+			Usage:  "git commit message",
+			EnvVar: "DRONE_COMMIT_MESSAGE",
+		},
+		cli.StringFlag{
+			Name:   "build.event",
+			Value:  "push",
+			Usage:  "build event",
+			EnvVar: "DRONE_BUILD_EVENT",
+		},
+		cli.IntFlag{
+			Name:   "build.number",
+			Usage:  "build number",
+			EnvVar: "DRONE_BUILD_NUMBER",
+		},
+		cli.StringFlag{
+			Name:   "build.status",
+			Usage:  "build status",
+			Value:  "success",
+			EnvVar: "DRONE_BUILD_STATUS",
+		},
+		cli.StringFlag{
+			Name:   "build.link",
+			Usage:  "build link",
+			EnvVar: "DRONE_BUILD_LINK",
+		},
+		cli.Int64Flag{
+			Name:   "build.started",
+			Usage:  "build started",
+			EnvVar: "DRONE_BUILD_STARTED",
+		},
+		cli.Int64Flag{
+			Name:   "build.created",
+			Usage:  "build created",
+			EnvVar: "DRONE_BUILD_CREATED",
+		},
+		cli.StringFlag{
+			Name:   "build.tag",
+			Usage:  "build tag",
+			EnvVar: "DRONE_TAG",
+		},
+		cli.Int64Flag{
+			Name:   "job.started",
+			Usage:  "job started",
+			EnvVar: "DRONE_JOB_STARTED",
+		},
 
-	if vargs.Port == 0 {
-		vargs.Port = 587
+		// .env file
+		cli.StringFlag{
```

Comment:
This won't work. If we want to load from an env file we have to do it like https://github.com/drone-plugins/drone-docker/blob/master/main.go#L16